package-scripts.js

```diff
@@ -21,8 +22,9 @@ module.exports = {
       script: 'git-cz',
     },
     test: {
-      default: 'jest --coverage',
-      watch: 'jest --watch',
+      default: crossEnv('NODE_ENV=test jest --coverage'),
+      update: crossEnv('NODE_ENV=test jest --coverage --updateSnapshot'),
+      watch: crossEnv('NODE_ENV=test jest --watch'),
```

Comment:
This is in prep for https://github.com/paypal/glamorous/issues/216

The `test.update` script will be really nice as well.